### DISCUSSION TITLE
Unable to complete publish on the e-notices2 platform

### DISCUSSION THREAD

**Discussion by Panagiotis-1** [OP-TED/eForms-SDK#858] [Q&A] [answered]
Good Morning Team,
Please help us with the problem we are facing when posting the notices on the e-notices2 platform.

Particularly:
While all the fields per category are correctly filled in and marked with the green symbol, they are not validated.
The paradox is that while e.g. the Process (Διαδικασία) category has been filled in correctly and all the fields have turned green and the Process (Διαδικασία) has indeed turned green in the vertical column, this is not preserved when we select the next category, e.g. LOT-0001 

When we go to continue e.g. LOT-0001 (ΤΜΗΜΑ 1), the Process changes and appears with the pencil symbol
 The same problem is observed in all the contest tabs.

As a result of the above, when we go to validate the document, the following message appears: 
«Your notification is not complete. Please check the empty mandatory fields».

The above message does not refer to the fields that have not been completed.

The same problem appears in every new competition we create.

We inform you that our notices have been completed in accordance with the guide issued by the Ministry of Development and we believe that no mandatory field is missing.

Please very much help us to understand the cause of the problem as we have already spent a lot of time and effort in training on the e-notices2 platform and now this particular problem has appeared for which we cannot find a solution and which forces us to request again your support.
Thank you very much in advance.

Kind regards
Panagiotis Tzoulias

**Comment by YvesJo** [comment, marked as answer]
Hi,
This is not an SDK issue, so it has been converted to a Q&A discussion. The discussion channel is also exclusively used as a forum for SDK related concerns, for issues with eNotices 2, please get directly in touch with the Support ted@publications.europa.eu and provide the XML in addition to any other relevant information (e.g. rule ID for the error message).
KR
